Makefile

```diff
@@ -29,7 +29,7 @@ upload: ## upload distribution package to PyPI
 
 .PHONY: upgrade-dist-tools
 upgrade-dist-tools: ## upgrade packages necessary for testing, building, packaging and uploading to PyPI
-	python3 -m pip install --upgrade pip setuptools wheel tqdm twine pytest mkdocs
```

Comment:
I removed tqdm from "upgrade-dist-tools", because it's used nowhere. Is that correct?

Reply:
tqdm is not part of dist tools, so no idea why this is there